.github/workflows/e2e-cross-version.yml

```diff
@@ -2,12 +2,31 @@ name: E2E Cross-version Tests
 
 on:
   schedule:
-    - cron: '0 9 * * 0'
+    - cron: '0 9 * * 1-5'
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
we want to run these on any pull request that touches the cross-version tests so that people know if they broke them